docs/content/server/setup.md

```diff
@@ -21,22 +21,23 @@ To setup the server, you need to have the following installed:
 - [Node.js](https://nodejs.org/en/)
 - [Docker](https://docs.docker.com/get-docker/)
 - [Git](https://git-scm.com/downloads)
+- [Yarn Classic](https://classic.yarnpkg.com/lang/en/docs/install/)
 
-### Checking if you have Node.js & NPM installed
+### Checking if you have Node.js & Yarn installed
 
 We use Node.js to run our frontend application.
 
 If you need assistance installing Node.js:
 https://nodejs.org/en/download/
 
-NPM is included with Node.js, but if you need assistance installing NPM:
-https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
+Yarn Classic can be installed easily through NPM (which is included with Node.js):
+https://classic.yarnpkg.com/lang/en/docs/install/
```

Comment:
Yarn Classic for now as migrating to modern Yarn is a bigger change that needs an announcement to prevent breaking DX. I've put out a [poll](https://discord.com/channels/1092243196446249134/1095817829405704305/1293993678766870682) on Discord to see whether Yarn Classic or modern Yarn is more popular right now, and based on the results I can hopefully make a better judgement whether we can switch without too much of a hassle.

Reply:
Apparently Corepack supports Yarn Classic as well, so I've added instructions to enable Corepack. This way we can upgrade to a newer version of Yarn without impact on DX.